src/netcdf_scm/iris_cube_wrappers.py

```diff
@@ -845,40 +846,43 @@ def _convert_scm_timeseries_cubes_to_openscmdata(
             scm_timeseries_cubes, out_calendar=out_calendar
         )
 
-        model, scenario = self._get_model_scenario()
+        climate_model, scenario = self._get_climate_model_scenario()
 
         out_df = pd.DataFrame(data, index=time_index)
         out_df.columns = pd.MultiIndex.from_product(
             [
                 [self.cube.standard_name],
                 [self.cube.units.name],
                 out_df.columns.tolist(),
-                [model],
+                [climate_model],
```

Comment:
Why? the IamDataFrame has a model field why not use that?

Reply:
the clash between their use of 'model' for an integrated assessment model and our use of 'model' for climate_model would become unwieldy, this is a workaround